### PR TITLE
renamed "spectral quantities" (radiance, transmittance, abscoeff, etc…

### PR DESCRIPTION
**Pull request by advide2001**
### Description

This pull request is to address "docs: rename "spectral quantities" (radiance, transmittance, abs coeff, etc.) to "spectral arrays" consistently throughout the code so we don't confuse them with "dimensioned quantities" / "units"?

Fixes part 2 of Issue #77

### CODE REVIEW

**Comment by erwanp** [comment]
Good ! Any better idea for the name ? 
@minouHub  we need your imagination :)

File: docs/spectrum/spectrum.rst [hunk 1]
```diff
@@ -158,7 +158,7 @@ Spectrum conditions and get Spectrum that suits specific parameters
 
 .. _label_spectral_quantities:
 
-Spectral quantities
+spectral arrays
 ===================
 
 A :class:`~radis.spectrum.spectrum.Spectrum` object can contain one spectral quantity, such as
```
Comment (minouHub) [inline docs/spectrum/spectrum.rst:164]: `A :class: ~radis.spectrum.spectrum.Spectrum object can contain one spectral **quantity**`

You missed one?

File: docs/spectrum/spectrum.rst [hunk 2]
```diff
@@ -158,7 +158,7 @@ Spectrum conditions and get Spectrum that suits specific parameters
 
 .. _label_spectral_quantities:
 
-Spectral quantities
+spectral arrays
```
Comment (minouHub) [inline docs/spectrum/spectrum.rst:161]: **S**pectral

**Review by minouHub — COMMENTED** [review]
(minor)
Capitalize + missed a "spectral quantity"

**Comment by minouHub** [comment]
Makes sense to me. I think we should not be too fancy on this name to avoid complication: it's an array, this array stores spectra. That's it.

**Comment by AlphaVictorDelta** [comment]
I'll make the required changes @minouHub.

File: docs/spectrum/spectrum.rst (hunk 1 above)
Comment (AlphaVictorDelta) [reply docs/spectrum/spectrum.rst:164]: I just realized it's not just on spectral quantity is in many different forms ( for, e.g., spectral_quantity, spectral quantities, etc.)

**Comment by erwanp** [comment]
@AlphaVictorDelta only the code linting tests are failing. Can you run `black` on your code? (see the Developer Guide for more information on code linting) 

Then we can merge!

**Comment by AlphaVictorDelta** [comment]
I will read up on it and resolve the issue.

**Comment by AlphaVictorDelta** [comment]
Something is really wrong with the changes I have made here. I will delete this pull request and create a brand new pull request.

**Comment by erwanp** [comment]
May be a Git error : did you merge (add a new merge commit)  the changes from the latest branch, where you should have rebased (insert changes in history, no new commit)  only?

**Pull request closed without merging**